### PR TITLE
Get rid of MaxListenersExceededWarning in tests

### PR DESCRIPTION
**Pull request by krzysztof-jelski**
# Checklist:

- [x] Code follows the style guidelines
- [x] Documentation updated
- [x] Corner cases evaluated and handled
- [x] Automatic tests added/update
- [x] Automatic tests pass
- [x] Software runs (locally or hosted)
- [x] Sanity check passed
- [x] Self-review passed

### CODE REVIEW

File: src/utils/web3_tools.js [hunk 1]
```diff
@@ -33,7 +33,9 @@ function createGanacheProvider(secretKey) {
       ...Array(9).fill({balance: '1000000000000000000000'})
     ]
   };
-  return Ganache.provider(ganacheOptions);
+  const provider = Ganache.provider(ganacheOptions);
+  provider.setMaxListeners(90);
```
Comment (Anze1m) [inline src/utils/web3_tools.js]:
```suggestion
  provider.setMaxListeners(500);
```
